### ISSUE TITLE
Conditional anyres-gan implementation failure

### ISSUE THREAD

**Issue by WenjinFu** [chail/anyres-gan#3]
I am working on generating conditional food images but met some errors on loading labels for patch data. I strictly follow the instructions in the file "train.sh" and "train.py". In the first stage of training, I first tried to leverage the pre-trained network of conditional stylegan3 to train the patch datasets in stage two and I got the following errors: 
![conditional img gen error](https://user-images.githubusercontent.com/57229263/178400814-af89f999-5fcc-4f83-b374-a91cc3aad289.jpg)
And it is caused by None type tensor as you can see from the terminal output messages. 
I read the paper, the authors use the unconditional image generation method. But I want to know if this method is compatible with conditional image generation.

**Comment by chail** [comment]
We only tried unconditional generation, but it should be compatible with the proper modifications. I suppose you'll need to modify the function `compute_scale_inputs`; in particular the `mapped_scale = G.scale_mapping(scale, None)`. Here I'm assuming that G.c_dim is None (for unconditional generation), but you could have it also take in a class latent for class-conditional generation.
